source/math/FloatMatrix.ooc

```diff
@@ -288,6 +301,58 @@ operator * (left: FloatMatrix, right: FloatMatrix) -> FloatMatrix {
 result
 }
 
+// <summary>
+// Addition of matrices.
+// </summary>
+// <param name="left">Left matrix in the addition.</param>
+// <param name="right">Right matrix in the addition.</param>
+// <returns>Sum of left and right matrices.</returns>
+operator + (left: FloatMatrix, right: FloatMatrix) -> FloatMatrix {
```

Comment:
That's odd, magic should have caught this...

`operator - (left, right: FloatMatrix) -> FloatMatrix {`